runner/client/src/mill/runner/client/JansiLoader.java

```diff
@@ -0,0 +1,50 @@
+package mill.runner.client;
+
+import java.io.File;
+import java.io.IOException;
+import org.fusesource.jansi.internal.OSInfo;
+
+final class JansiLoader {
+
+  private String jansiVersion;
+
+  JansiLoader(String jansiVersion) {
+    this.jansiVersion = jansiVersion;
+  }
+
+  private String jansiLibPathInArchive() {
+    return "org/fusesource/jansi/internal/native/" + OSInfo.getNativeLibFolderPathForCurrentOS()
+        + "/" + System.mapLibraryName("jansi").replace(".dylib", ".jnilib");
```

Comment:
This `String#replace` call looks a bit weird; maybe add a comment explaining why we're doing this?

Reply:
Just did that